blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
 image: 
 category: 3
 summary: |
-  This blueprint will help you use the Genesys Bot Connector to connect to a Microsoft Power Virtual Agent to answer customer queries via Web Messaging. It also provides a guide to do this for any other bot technology that Genesys does not support as a strategic vendor.
+  This Genesys Cloud Developer Blueprint explains how to deploy a Microsoft Power Virtual Agent (VA) bot in Genesys Cloud. The blueprint solution uses Genesys Bot Connector that provides the API to connect to the bot. The Genesys Bot Connector acts as the link between Genesys Cloud and the bot.
 ---
 
-This Genesys Cloud Developer Blueprint provides instructions for deploying a Microsoft Power VA bot in Genesys Cloud. The Genesys Bot Connector is used here to provide an API for the bot to connect to, but a direct link between Genesys Cloud and the bot is not possible. For this scenario to work, an application (known as the Bot Interpreter and represented by the Lambda function in the diagram) needs to be in between Genesys Cloud and Power VA. This application takes the message from the customer using Messenger (known as the utterance) from Genesys Bot Connector and changes the format before sending out a HTTP request to Power VA for NLU. The return message is also converted by the Bot Interpreter to match the postUtterance API provided by Genesys Bot Connector.
+This Genesys Cloud Developer Blueprint explains how to deploy a Microsoft Power Virtual Agent (VA) bot in Genesys Cloud. The blueprint solution uses Genesys Bot Connector that provides the API to connect to the bot. The Genesys Bot Connector acts as the link between Genesys Cloud and the bot. 
 
-![Flowchart for the bot connector solution](images/flowchart_bot_connector.png "Flowchart for the bot connector solution")
+## Scenario
 
+Customers might use different third-party chatbots in their business. Genesys Cloud supports and provides interface systems for selected chatbot providers such as AWS and Google. Many new chatbot vendors are fast-growing in the market. Customers want to bring in third-party bots in the Architect message flows and populate the bot list.
+
+## Solution
+Genesys Bot Connector provides the API to call third-party bots in an Architect message flow. The blueprint showcases how to connect to the Microsoft Power VA bot in Genesys Cloud using the AWS services:
+
+1. Create an application using AWS Lambda functions. The application known as the Bot Interpreter acts in between Genesys Cloud and Power VA. 
+2. Create a chatbot with topics and entities in Microsoft Power VA.
+3. Configure a Messenger in your website.
```

Comment:
Start with "In your website,..." 
Start by telling the user where they will complete the work. In blueprints it's especially important since we are switching contexts quickly.